csatar-plugins/csatar/models/Scout.php

```diff
@@ -50,7 +53,10 @@ class Scout extends Model
         'legal_relationship' => '\Csatar\Csatar\Models\LegalRelationship',
         'special_diet' => '\Csatar\Csatar\Models\SpecialDiet',
         'religion' => '\Csatar\Csatar\Models\Religion',
-        'tshirt_size' => '\Csatar\Csatar\Models\TShirtSize'
+        'tshirt_size' => '\Csatar\Csatar\Models\TShirtSize',
+        'team' => '\Csatar\Csatar\Models\Team',
+        'troop' => '\Csatar\Csatar\Models\Troop',
```

Comment:
A Troop es Patrol modellekbe "hasMany"-be betenni a Scout-ot.

Reply:
Betéve.